### ISSUE TITLE
Ubuntu unity system tray menu

### ISSUE THREAD

**Issue by ghost** [dashpay/dash#1332]
Menu is missing from the system tray in Ubuntu (16.04), it just shows as an empty rectangle approx. 8 lines long and 8 char. wide.

**Comment by crowning-** [comment]
Do you mean the 'normal' menus on the left side of the tray?

Which desktop (Unity I guess) are you using?

**Comment by stan-distortion** [comment]
Yes, Unity. The menu on the left side is ok (limited but the menu is showing up fine), it's the one in the system tray in the top right. Maybe close as it appears to be a Unity issue:
https://bugs.launchpad.net/ubuntu/+source/bamf/+bug/1532226

Only difference is the suggested workaround in there ("initctl restart unity-panel-service") doesn't correct it, the icon disappears entirely from the system tray instead. Sorry for the delay, wanted to try a few things before reporting back and got held up.

**Comment by t-nelson** [comment]
Any motion here?

I just updated from 0.12.0.53 to 0.12.1.5 (built from the git tag) and am seeing similar.  In my case the tray icon renders properly, but is not in the Unity panel/systray (whatever they call it now).  Instead, it is always positioned in the far top-left of my left-most monitor.

The "Minimize to the tray instead of the taskbar" setting seems to have no effect.  Nor does changing themes.  Nothing obvious in any of the logs.  If I can find some time, I'll try to dig around in the code.

Ubuntu 14.04.5 LTS - fully patched.
Unity 7.2.6
Qt 5.4.1
I built with all deps from the standard Ubuntu Trusty repos, except for Berkely DB 4.8, which was from Oracle.

Let me know if I can provide any further info.
